### PR TITLE
New/brotli

### LINKED ISSUE #1604: [Feature] Migrate brotli to built-in implementation
# 🚀 Feature request

## Description

Node will soon have built-in support for brotli per:
https://github.com/nodejs/node/pull/24938
https://twitter.com/addaleax/status/1081653502693257217?s=21

It will be great to migrate our usage to this version.

We will have to see where it lands (node 11 or maybe it gets back ported) and work on a planif it gets back ported to previos versions of node and if not work on a plan to remove our current dependencies in favor of this one.

 Are you willing to work on this?
Yup

## What scenarios will this solve?

Less dependencies to download and keep up to date. Also we had problems with the binaries in the past so less risk.

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~~Added/Updated related documentation.~~
- ~~Added/Updated related tests.~~

## Short description of the change(s)

Fix #1604

### CODE REVIEW

**Comment by molant** [comment]
Not sure when we updated the docs, but we are already recommending node 10

File: packages/configuration-accessibility/package.json [hunk 1]
```diff
@@ -6,9 +6,6 @@
     "@hint/hint-axe": "^4.1.6"
   },
   "description": "webhint's axe-based accessibility configuration for WCAG 2.1 Level A and Level AA requirements",
-  "engines": {
```
Comment (sarvaje) [inline packages/configuration-accessibility/package.json]: Doesn't we need this anymore?

Comment (molant) [reply packages/configuration-accessibility/package.json]: I'm leaving it only on entry packages (`hint`, `create-*`) so next time we drop support for a node version we don't have to touch everything.

Comment (molant) [reply packages/configuration-accessibility/package.json]: Although I should probably update the `create-*` templates

**Comment by molant** [comment]
This PR has been rebased. @antross @sarvaje can you please take a look?

**Comment by molant** [comment]
Although I'm tempted to merge this right now, I'll wait until #3176 gets merged first. Sounds good @sarvaje? 😛

**Comment by molant** [comment]
I've rebased this branch and will be merging once it passes CI

**Pull request closed without merging**